maintainers/maintainer-list.nix

```diff
@@ -8027,6 +8027,12 @@
     githubId = 854319;
     name = "Matt McHenry";
   };
+  jerrysm64 = {
+    email = "jerry.starke@icloud.com";
+    github = "jerrysm64";
+    githubID = "42114389";
```

Comment:
```suggestion
    githubId = "42114389";
```

Reply:
Just fixed it. Can you double check again, please?

Reply:
LGTM, but I don’t have write permission and need to wait for committers to approve